src/main/rx/rx.c

```diff
@@ -746,14 +758,14 @@ uint8_t getRssiPercent(void)
 }
 
 #ifdef USE_RX_LINK_QUALITY_INFO
-uint8_t rxGetLinkQuality(void)
+uint16_t rxGetLinkQuality(void)
 {
     return linkQuality;
 }
 
-uint8_t rxGetLinkQualityPercent(void)
+uint16_t rxGetLinkQualityPercent(void)
 {
-    return scaleRange(rxGetLinkQuality(), 0, LINK_QUALITY_MAX_VALUE, 0, 100);
+    return (linkQualitySource == LQ_SOURCE_RX_PROTOCOL_CRSF) ?  (linkQuality / 3.41) : scaleRange(linkQuality, 0, LINK_QUALITY_MAX_VALUE, 1, 99);
```

Comment:
Why is this now limited to 99?

Reply:
will fix   ops...
rssi display is 0-99 but stats are 0-100 .
and the original LQ was 0-9 but stats  0-100% ..  

users would be asking where is the 1% of packet loss..  and i would be looking fr a hole to climb into .